pkg/pillar/cmd/zedagent/reportinfo.go

```diff
@@ -659,11 +660,10 @@ func PublishAppInstMetaDataToZedCloud(ctx *zedagentContext, appInstID string, ap
 	ReportInfo.AtTimeStamp = ptypes.TimestampNow()
 
 	ReportAppInstMetaData := new(info.ZInfoAppInstMetaData)
-	ReportAppInstMetaData.Uuid = appInstID
+	ReportAppInstMetaData.Uuid = pubsubKeyStr
```

Comment:
Here is where it gets set in the API. That host remain the same UUID as before.

Reply:
+1